Godeps/_workspace/src/github.com/openshift/openshift-sdn/ovssubnet/registry/registry.go

```diff
@@ -30,27 +30,42 @@ type EtcdSubnetRegistry struct {
 	etcdCfg *EtcdConfig
 }
 
-func newMinionEvent(action, key, value string) *api.MinionEvent {
-	min := &api.MinionEvent{}
+func newNodeEvent(action, key, value string) *api.NodeEvent {
+	nodeEvent := &api.NodeEvent{}
 	switch action {
 	case "delete", "deleted", "expired":
-		min.Type = api.Deleted
+		nodeEvent.Type = api.Deleted
 	default:
-		min.Type = api.Added
+		nodeEvent.Type = api.Added
 	}
 
 	if key != "" {
-		_, min.Minion = path.Split(key)
-		return min
+		_, nodeEvent.Node = path.Split(key)
+
+		var node map[string]interface{}
```

Comment:
This deserialization is for the standalone mode of openshift-sdn (not shipped/supported for enterprise).
It does not get used in the openshift product. The actual registry used in openshift is in <origin>/plugins/osdn/